src/widgets/VaccineChart.js

```diff
@@ -155,40 +230,78 @@ export class VaccineChart extends React.Component {
   }
 }
 
+const chartSize = {
```

Comment:
Only change I actually will request: Should use `react-native`  class `StyleSheet` for these.

```
StyleSheet.create({
    chartStyles: {
    ...
    },
})
```

Not only is it consistent with the codebase, but it also increases performance as styles can be referred to be reference (styles will be assigned an ID) and also *I think* will be cached

Reply:
Fixed by cba7baac (some of the styles are victory native props for styling, but are victory styles not CSS styles, which `Stylesheet.create` does not seem to like).